### ISSUE TITLE
Readme needs clarity WRT how to test bundles locally

### ISSUE THREAD

**Issue by resource11** [CLIxIndia-Dev/unplatform_v2#100]
We need to add more clarity with respect to how a user can test their bundle, i.e., what steps need to happen to run the built app.

**Comment by cjshawMIT** [comment]
Something like the "testing" section of this doc?

https://docs.google.com/document/d/1JOk6cFT8g7N5EahJ4Ss_QmVZ9PjYYMTIFbDiKkxzZsA/edit#heading=h.h0cxgrv4k4tm

**Comment by resource11** [comment]
Yes! We can do a wiki to keep the readme more concise with the basic setup and dive deeper in a wiki.

**Comment by cjshawMIT** [comment]
I've migrated the documentation to this page:

https://github.com/CLIxIndia-Dev/unplatform_v2/wiki/Verifying-the-Bundle

If that looks good, please close this issue.

**Comment by resource11** [comment]
Thanks for migrating that over. I've looked at the wiki, and it gives a much clearer explanation of how to complete many of the essential steps. 

On cursory glance, I think it'd be of value to:
1. Add a few extra details for the layperson. Most are in there, simply needs a bit more clarity in places
2. Be sure all hyperlinks map to correct locations. For example, the hyperlinks to OEA sometimes point to our fork, sometimes to the Atomic Jolt fork.
3. Either rearrange some of the content flow, or add copies of chunks of content from certain pages of the wiki to also appear in other parts of the wiki to reduce the amounts of clicks to gather info.

**Comment by cjshawMIT** [comment]
It seems like some of your comments refer to the wiki in general, not to that specific page that I linked to, that addresses this issue. So:

- Can you specify where extra detail would help? That would be simpler for me to start on, since I don't have the layperson filter. Especially if the unclear areas are on a different page, it would be good to know which one.
- I only saw one link to AtomicJolt, and I've updated that to point to our fork (the link on the `Home` page that said the tool works with `open_assessments`). But you said "hyperlinks" -- can you please point to the other ones?
- Can you provide more specifics on which content areas could be reworked? More specific feedback would help me clarify, otherwise I risk muddying the waters even more.

**Comment by resource11** [comment]
Ahh, yes, it's more generalized comments. For many of the details, I can sync with you in person to go over which areas could use the layperson filter. 

I see you fixed the AtomicJolt link, the other one I found so far is more of a need to change a repo's url to map to the actual tool name (the Run Kitty Run repo's url is still listing the old tool's name) in the Bundling Unplatform wiki page.

Some of the lowest-hanging fruit to tackle in the meantime, though, is making sure links more clear to the actual location to where they are pointing. 

For example, in the Bundling Unplatform wiki, this sentence:

To learn more about virtual environments, you can read more **here** (linked to the Hitchhiker's Guide) 

may read more clearly to the user as this:

To learn more about virtual environments including how to set one up, you can read more at the **Hitchhiker's Guide to Python**.

**Comment by cjshawMIT** [comment]
Okay, discussing the confusing parts face-to-face sounds good.

I forgot about Run Kitty Run, thanks for finding that. I also updated the section about the build script, which was updated. I'll go through and clean up the links.

Thanks!

**Comment by resource11** [comment]
Sounds great! I found some decent examples of well-structured repos with respect to documentation style that we can look at for best practices.
